google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/BigQueryImplTest.java

```diff
@@ -1134,39 +1183,45 @@ public void testQueryRequestCompleted() throws InterruptedException {
     Map<BigQueryRpc.Option, Object> optionMap = Maps.newEnumMap(BigQueryRpc.Option.class);
     QueryResultsOption pageSizeOption = QueryResultsOption.pageSize(42L);
     optionMap.put(pageSizeOption.getRpcOption(), pageSizeOption.getValue());
+
     EasyMock.expect(
-        bigqueryRpcMock.getQueryResults(PROJECT, JOB, optionMap)).andReturn(responsePb);
+            bigqueryRpcMock.getQueryResults(
+                PROJECT, JOB, BigQueryImpl.optionMap(Job.DEFAULT_QUERY_WAIT_OPTIONS)))
+        .andReturn(responsePb);
+    EasyMock.expect(bigqueryRpcMock.listTableData(PROJECT, DATASET, TABLE, optionMap))
+        .andReturn(
+            new TableDataList()
+                .setPageToken("")
+                .setRows(ImmutableList.of(TABLE_ROW))
+                .setTotalRows(1L));
 
     EasyMock.replay(bigqueryRpcMock);
     bigquery = options.getService();
-    QueryResponse response = bigquery.query(
-        QUERY_JOB_CONFIGURATION_FOR_QUERY,
-        queryJob,
-        QueryOption.of(QueryResultsOption.pageSize(42L)));
-    assertNull(response.getEtag());
-    assertEquals(queryJob, response.getJobId());
-    assertEquals(true, response.jobCompleted());
-    assertEquals(false, response.getResult().cacheHit());
-    assertEquals(ImmutableList.<BigQueryError>of(), response.getExecutionErrors());
-    assertFalse(response.hasErrors());
-    assertEquals(null, response.getResult().getSchema());
-    assertEquals(42L, response.getResult().getTotalBytesProcessed());
-    assertEquals(1L, response.getResult().getTotalRows());
-    for (FieldValueList row : response.getResult().getValues()) {
-      assertEquals(false, row.get(0).getBooleanValue());
-      assertEquals(1L, row.get(1).getLongValue());
+    // TODO(pongad): pagesize = 42
```

Comment:
Orphan TODO? I don't have enough context for why this is here, so if it stays it should have more info.

Reply:
This TODO has been resolved, since we now properly pass page size here to `getQueryResults`.

The TODO below is deleted without resolving. Currently there's no way to set page size by `query()` method. I think it's OK to provide fewer options to "short hand" commands.